alcs-frontend/src/styles/functions.scss

```diff
@@ -0,0 +1,10 @@
+@use 'sass:math';
+
+@function rem($pixels, $context: 16) {
```

Comment:
The reason that we did not put rem function in alcs-frontend was that alcs-frontend would be only used on desktop devices so rem would not be useful.